### ISSUE TITLE
V5.0.0 - Rest api groups.listAll endpoint in timeout

### ISSUE THREAD

**Issue by CodazziS** [RocketChat/Rocket.Chat#26418]
### Description:

Since v5.0.0, when I try to call groups.listAll (or channels.list), the request timeout.

I have no result error, no error logs in rocketchat.

### Steps to reproduce:

Curl command:
```curl --request GET \
  --url http://localhost:3000/api/v1/channels.list \
  --header 'X-Auth-Token: xxx' \
  --header 'X-User-Id: xxx'
  ```

### Expected behavior:

Get the groups list.

### Actual behavior:

The result command after 120 secondes is : 
`curl: (52) Empty reply from server`

### Server Setup Information:

- Version of Rocket.Chat Server: 5.0.0
- Operating System: Ubuntu 22.04
- Deployment Method: Docker swarm
- Number of Running Instances: 1
- DB Replicaset Oplog: 
- NodeJS Version: 14.19.3
- MongoDB Version: 5.0.10 

### Client Setup Information

- Desktop App or Browser Version: NA
- Operating System: NA

### Additional context

Work fine with fresh dev installation, but cannot find why my production can broke the rest API.

**Comment by kiwi-oss** [comment]
It seems to be [fixed in 5.0.3](https://github.com/RocketChat/Rocket.Chat/issues/26483#issuecomment-1212770909).

**Comment by ggazzo** [comment]
closed by https://github.com/RocketChat/Rocket.Chat/pull/26530
